src/system/sys_profiler.hh

```diff
@@ -524,6 +638,25 @@ public:
      */
     inline double actual_cumulative_time() const
         { return timers_[actual_node].cumulative_time(); }
+    /**
```

Comment:
They are used in unit tests. We can't directly access Timer values since they are private.